app/models/local_authority.rb

```diff
@@ -18,21 +18,39 @@ class LocalAuthority < ResponsibleBody
 
   has_one :la_funded_place, foreign_key: 'responsible_body_id'
 
-  def create_la_funded_places!(urn:, device_allocation: 0, router_allocation: 0, extra_args: {})
+  def create_iss_places!(device_allocation: 0, router_allocation: 0, extra_args: {})
+    create_la_funded_places!(urn: "ISS_#{gias_id}",
+                            name: 'State-funded pupils in independent special schools and alternative provision',
+                            device_allocation: device_allocation,
+                            router_allocation: router_allocation,
+                            extra_args: extra_args)
+  end
+
+  def create_scl_places!(device_allocation: 0, router_allocation: 0, extra_args: {})
```

Comment:
actually can we remove this method for this PR. I think having the same instance type of `LaFundedPlace` for Social Care Leavers will be problematic as that just leaves the name of the school as the only anchor point to hook and attach behaviour to which I think is quite brittle. I'd rather introduce another type to remove any ambiguity

Reply:
Updated to use `provision_type` on `LaFundedPlace` to differentiate